client/reader/search-stream/style.scss

```diff
@@ -63,17 +69,12 @@
 	top: 0;
 	z-index: 20;
 	-webkit-font-smoothing: subpixel-antialiased; // Fixes fixed elements text aliasing in Safari
-	padding: 0 1px;
-    margin: 0 -1px;
-	@media only screen and (min-width: 601px) and (max-width: 660px) {
-		width: calc(100% - 96px);
-	}
-}
+	padding: 24px 0 0;
+	margin: 0 -1px;
 
-.layout.has-header-section .search-stream__fixed-area {
-	position: static;
-	padding-top: 0;
-	margin-top: 0;
+	.search__input-fade::before {
+		border-radius: 4px;
```

Comment:
Fixes missing top right and bottom right bottom radius.

![image](https://github.com/user-attachments/assets/89b76261-d56a-45e1-b96b-3db71879db98)

Reply:
Nice catch!